src/ert/config/queue_config.py

```diff
@@ -37,7 +37,7 @@ def __post_init__(self) -> None:
             setting
             for settings in self.queue_options.values()
             for setting in settings
-            if isinstance(setting, tuple) and setting[0] == "MAX_RUNNING"
+            if setting[0] == "MAX_RUNNING" and setting[1]
```

Comment:
Will probably work, but do we need that extra for-loop here after the union was removed?
Also this seems a bit excessive way to test this one option, can this be refactored somehow?

Reply:
I put this check into the main for-loop, but unsure if this improves anything. The validations are all over the place in this class.

Reply:
I believe that with the current structure of `queue_options`, the extra for loop is needed to unpack all values correctly from .values().